Calling/ClientApp/src/components/Configuration.tsx

```diff
@@ -78,9 +78,12 @@ export default (props: ConfigurationScreenProps): JSX.Element => {
 
   const { groupId, setupCallClient, setGroup, unsupportedStateHandler } = props;
 
+  const memoizedSetupCallClient = useCallback(() => setupCallClient(unsupportedStateHandler), [
+    unsupportedStateHandler
```

Comment:
Please include `setupCallClient` in the dependency array. You might get stale props otherwise.